README.md

```diff
@@ -224,3 +224,73 @@ Thanks also give to these sponsors or backers:
 ## 🪪 License
 
 [MIT](https://github.com/js-sdsl/js-sdsl/blob/main/LICENSE) © [ZLY201](https://github.com/zly201)
+
+[stack-package]: https://github.com/js-sdsl/js-sdsl/blob/main/src/container/OtherContainer/Stack.ts
+[stack-npm-version]: https://img.shields.io/npm/v/@js-sdsl/stack
+[stack-npm-link]: https://www.npmjs.com/package/@js-sdsl/stack
+[stack-umd-size]: https://img.badgesize.io/https://unpkg.com/@js-sdsl/stack/dist/umd/stack.min.js?compression=gzip&style=flat-square/
+[stack-umd-link]: https://unpkg.com/@js-sdsl/stack/dist/umd/stack.min.js
+[stack-docs]: https://js-sdsl.org/js-sdsl/classes/Stack.html
+
+[queue-package]: https://github.com/js-sdsl/js-sdsl/blob/main/src/container/OtherContainer/Queue.ts
+[queue-npm-version]: https://img.shields.io/npm/v/@js-sdsl/queue
+[queue-npm-link]: https://www.npmjs.com/package/@js-sdsl/queue
+[queue-umd-size]: https://img.badgesize.io/https://unpkg.com/@js-sdsl/queue/dist/umd/queue.min.js?compression=gzip&style=flat-square/
+[queue-umd-link]: https://unpkg.com/@js-sdsl/queue/dist/umd/stack.min.js
```

Comment:
fix to:
[queue-umd-link]: https://unpkg.com/@js-sdsl/queue/dist/umd/queue.min.js

Reply:
done